examples/module2/lesson2/extended/tests/e2e/logged-in/search-term-in-faq.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '../../../fixtures';
+import { MainPage } from '../../../pages/main.page';
+import { URLs } from '../../../utils/constants';
+import { SearchQuestionsPage } from '../../../pages/search-questions.page';
+import { ResultsListPage } from '../../../pages/results-list.page';
+
+test('search term in faq section', async ({ page }) => {
```

Comment:
Rozważyłbym rozwinięcie nazwy tego testu na taką, która wskazuje na to, co sprawdzamy.